### PR TITLE
more contrast changes

### PR DESCRIPTION
**Pull request by brittyazel**
Focusing on small details in the main panel and admin panels, these changes just make a few contrast tweaks to add to the overall look

### CODE REVIEW

**Comment by pbaity** [comment]
@brittyazel Request my review or mention me when this PR is ready to be merged.

**Comment by brittyazel** [comment]
Pr is ready! Happy for a review

File: dark-mode.css [hunk 1]
```diff
@@ -336,6 +342,22 @@ body.dark-mode .permissions-manager .permission-grid .id-styler {
 	color: var(--info-font-color);
 }
 
+body.dark-mode .rc-header--room {
+	box-shadow: 0px 0px 3px var(--color-gray-medium);
```
Comment (pbaity) [inline dark-mode.css]: In light mode, the box shadow on this div is pretty subtle, but this setting is pretty bold. What would you think of either thinning the shadow or darkening the color?

Here's the current look:
image

Here's with `box-shadow: 0px 0px 1px var(--color-gray-medium);`:
image

And here's with `box-shadow: 0px 0px 3px var(--color-dark-medium);`:
image

Let me know what you and your users think.

**Comment by brittyazel** [comment]
Hrm, I had tried the 2nd look first off, but I felt it looked too sharp and needed s bit more feathering, which is why I opted for the 3px blur on the current one. The last one is quite nice and I like the subtleness

File: dark-mode.css (hunk 1 above)
Comment (brittyazel) [reply dark-mode.css]: Hrm, I had tried the 2nd look first off, but I felt it looked too sharp and needed a bit more feathering, which is why I opted for the 3px blur on the current one. The last one is quite nice and I like the subtleness. I hadn't thought to just darken the color, which is brilliant

Comment (brittyazel) [reply dark-mode.css]: Yeah, after testing I wholeheartedly love option 3

Comment (pbaity) [reply dark-mode.css]: Great! Unless you want to try any different shades, you can accept this suggestion:

```suggestion
	box-shadow: 0px 0px 3px var(--color-dark-medium);
```

**Comment by brittyazel** [comment]
Did that work?

**Comment by pbaity** [comment]
@brittyazel That worked, and your changes look even better than before! Ready to merge?

**Comment by brittyazel** [comment]
Yessir!

**Comment by brittyazel** [comment]
btw, word on the street is that RocketChat is moving to using the React Native app for all apps, including the web app, moving forward. So this code may end up being non-compatible with a React version in a future update. The good news is the React version has a dark mode built in.

**Comment by pbaity** [comment]
As long as there's a dark mode available, I'll be satisfied. Merging your changes now.

**Pull request merged**